Classes/Common/Solr.php

```diff
@@ -578,8 +576,10 @@ protected function __construct($core)
                     'scheme' => $solrInfo['scheme'],
                     'host' => $solrInfo['host'],
                     'port' => $solrInfo['port'],
-                    'path' => '/' . $solrInfo['path'] . '/',
+                    'path' => '/',
                     'core' => $core,
+                    // For Solr Cloud you need to provide a collection instead of core:
+                    // 'collection' => $core
```

Comment:
That's good to know and a nice addition, but currently we don't support Solr Cloud. If we do, we have to change the way we automatically create new cores as well.